WordPress/gutenberg
# [RNMobile] Use multiple sliders in column parent

## Description

That PR introduces multiple sliders in the column's parent to control width of all inner columns.



## How has this been tested?

1. Open mobile app
2. Add columns block
3. Press `Cancel` button
4. Open columns settings
5. Play with sliders
**6. Expect that fill level in the preview (to the left of a slider) is changing proportionally in all columns** 
 






## Screenshots 
<img width="300" src="https://user-images.githubusercontent.com/22746080/89766881-27f06800-daf9-11ea-86b8-1ce17c5ee9d0.gif" />

## Types of changes

New feature





## Checklist:
